src/components/ProfilePage/AgeMessage.jsx

```diff
@@ -1,16 +1,30 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 import { UncontrolledAlert } from 'reactstrap';
+import { FormattedMessage } from 'react-intl';
 
 function AgeMessage({ accountURL }) {
   return (
     <UncontrolledAlert color="info">
-      <h6>Your profile cannot be shared.</h6>
-      <p>
-        To share your profile with other edX learners,
-        you must confirm that you are over the age of 13.
-      </p>
-      <a href={accountURL}>Set your date of birth</a>
+      <FormattedMessage
+        id="profile.age.headline"
+        defaultMessage="Your profile cannot be shared."
+        description="error message"
+        tagName="h6"
+      />
+      <FormattedMessage
+        id="profile.age.details"
+        defaultMessage="To share your profile with other edX learners, you must confirm that you are over the age of 13."
+        description="error message"
+        tagName="p"
+      />
+      <a href={accountURL}>
+        <FormattedMessage
+          id="profile.age.set.date"
+          defaultMessage="Set your date of birth"
+          description="label on a link to set birthday"
+        />
+      </a>
```

Comment:
Oh boy this makes our markup hard to read :(

Reply:
If you get sick of it, you can pull it out to a .messages.jsx file... maybe we'll standardize on something later.